pxr/usdImaging/usdviewq/stageView.py

```diff
@@ -2101,9 +2120,14 @@ def mousePressEvent(self, event):
                 self.switchToFreeCamera()
                 ctrlModifier = event.modifiers() & QtCore.Qt.ControlModifier
                 self._cameraMode = "truck" if ctrlModifier else "tumble"
-            if event.button() == QtCore.Qt.MidButton:
-                self.switchToFreeCamera()
-                self._cameraMode = "truck"
+            if PySideModule == 'PySide6':
+                if event.button() == QtCore.Qt.MiddleButton:
+                    self.switchToFreeCamera()
+                    self._cameraMode = "truck"
+            else:
+                if event.button() == QtCore.Qt.MidButton:
+                    self.switchToFreeCamera()
+                    self._cameraMode = "truck"
```

Comment:
Seems we are already using `MiddleButton` in frameSlider.py, so it is available in pyside2, so you can just "upgrade" here unconditionally!

Reply:
Cheers